glean-core/android/src/main/java/mozilla/telemetry/glean/Glean.kt

```diff
@@ -377,6 +377,25 @@ open class GleanInternalAPI internal constructor() {
         return gleanTestGetExperimentationId() ?: throw NullPointerException("Experimentation Id is not set")
     }
 
+    /**
+     * Register a listener to receive event recording notifications
+     *
+     * @param tag a tag to use when unregistering the listener
```

Comment:
What happens if a second listener is registered with a previous tag?

Reply:
It would be hijacked, I suppose. I don't expect anyone other than Nimbus to use this (famous last words...), so I really don't _think_ this will be a problem (more famous last words...).

Reply:
Then we should document it. "Only one listener may be registered for a given tag. Each subsequent registration with a given tag unregisters the previous listener for that tag" or whatever.